src/Interpreters/ActionsDAG.cpp

```diff
@@ -624,7 +624,7 @@ ActionsDAGPtr ActionsDAG::makeConvertingActions(
             {
                 auto & input = inputs[res_elem.name];
                 if (input.empty())
-                    throw Exception("Cannot find column " + backQuoteIfNeed(res_elem.name) + " in source stream",
+                    throw Exception("Cannot find column '" + backQuote(res_elem.name) + "' in source stream",
```

Comment:
Do we need another one pair quotes here?
Or maybe we need it in other changed exception messages of this file?